README.md

```diff
@@ -1,193 +1,233 @@
-directory until they are written out to GCS due to being closed or fsync'd.
-Therefore the user must ensure that there is enough free space available to
-handle staged content when writing large files.
-
-## Other performance issues
-
-If you notice otherwise unreasonable performance, please [file an
-issue][issues].
-
-[issues]: https://github.com/googlecloudplatform/gcsfuse/issues
+Using the [Cloud Storage FUSE CSI driver](https://github.com/GoogleCloudPlatform/gcs-fuse-csi-driver), users get the declarative nature of Kubernetes
+with all infrastructure fully managed by GKE in combination with Cloud Storage. This CSI
+driver relies on Cloud Storage FUSE to mount Cloud storage buckets as file systems on the
+GKE nodes, with the Cloud Storage FUSE deployment and management fully handled by GKE, 
+providing a turn-key experience
 
 # Support
 
-gcsfuse is open source software, released under the [Apache license](LICENSE).
-It is distributed as-is, without warranties or conditions of any kind.
-
-For support, visit [Server Fault][sf]. Tag your questions with `gcsfuse` and
-`google-cloud-platform`, and make sure to look at
-[previous questions and answers][previous] and [troubleshooting.md](docs/troubleshooting.md) 
-before asking a new one. For bugs and feature requests, please [file an issue][issues].
-
-[sf]: http://serverfault.com/
-[previous]: http://serverfault.com/questions/tagged/gcsfuse
-
-# Versioning
-
-gcsfuse version numbers are assigned according to [Semantic
-Versioning][semver]. Note that the current major version is `0`, which means
-that we reserve the right to make backwards-incompatible changes.
-
-[semver]: http://semver.org/
\ No newline at end of file
+## Supported applications and platforms
+
+Cloud Storage FUSE officially supports  the latest two versions of the following 
+Machine Learning frameworks:
+
+- TensorFlow V2.x
+- TensorFlow V1.x
+- PyTorch V1.x
+
+Support outside of the above Machine Learning use cases is non-guaranteed, and on 
+a best effort basis.
+
+## Supported operating systems
+
+Cloud Storage FUSE supports usage with the following operating systems:
+- Ubuntu 18.04 and above
+- Debian 10 and above
+- CentOS 7 and above
+- RHEL 7 and above
+
+## Getting support
+
+Request support via your regular Google Cloud support channels, or by opening an
+issue via GitHub [here](https://github.com/GoogleCloudPlatform/gcsfuse/issues)
+
+See [Troubleshooting] for common issue handling
+
+[Troubleshooting]:https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/troubleshooting.md
+
+## Limitations and restrictions
+
+Cloud Storage FUSE is not supported for usage with the following workloads or scenarios:
+
+- Workloads that expect POSIX compliance.
+     - Cloud Storage FUSE is not a traditional POSIX filesystem. For a fully supported
+       enterprise grade filesystem, see [Cloud Filestore]. See Key Differences from a 
+       POSIX filesystem and Semantics doc
```

Comment:
Can you please provide the link for key-difference from a traditional file-system.

Reply:
fixed